core/src/main/java/io/aiven/klaw/model/requests/AclRequestsModel.java

```diff
@@ -40,9 +40,7 @@ public class AclRequestsModel extends BaseRequestModel implements Serializable {
   private String transactionalId;
 
   // topic owner team id
-  private Integer teamId; // OPENAPI add notnull
-
-  private String teamname; // OPENAPI Remove
+  @NotNull private Integer teamId;
```

Comment:
@aindriu-aiven here is where @NotNull is added to teamId.  github doesn't show that well

Reply:
Yeah I got that I was just wondering why we were deleting the String teamname